### PR TITLE
Ianjennings/config refactor

### PR DESCRIPTION
**Pull request by ianjennings**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -11,39 +11,11 @@ import { Root } from "./styles";
...
-  let result = { plugins: [], presets: [] };
-  result.plugins = babelConfig.plugins?.map(plugin => {
-    return {
-      name: plugin[0],
-      description: plugins[plugin[0]].description,
-      fileLocation: plugins[plugin[0]].fileLocation,
-      defaultConfig: plugin[1],
-    };
-  });
-}
```
Comment (garrettluu) [inline src/components/App.js:46]: Rest in peace 😭

File: src/components/App.js [hunk 2]
```diff
@@ -140,6 +112,38 @@ export const App = ({ defaultSource, defaultConfig, defCustomPlugin, defaultId,
   importDefaultPlugins();
   registerDefaultPlugins();
 
+  useEffect(() => {
+
+    setPanes(jsonConfig.map((config, index) => {
+
+      return {
+        menuItem: 'Config ' + index, render: () =>
+          <><Output
```
Comment (garrettluu) [inline src/components/App.js:124]: What does the `<>` do?

**Review by garrettluu — APPROVED** [review]
Looks good to me!

File: src/components/App.js (hunk 2 above)
Comment (ianjennings) [reply src/components/App.js:124]: It's a "react fragment" shorthand:
https://reactjs.org/docs/fragments.html#short-syntax

**Pull request merged**